### ISSUE TITLE
Youtube videos shows up as unavaliable

### ISSUE THREAD

**Issue by necroshadow0** [kamranahmedse/pennywise#62]
This is an issue. When I opened up Pennywise, opened up a vid link, and it proceeded to say the video was unavailable. I rebooted the app, refreshed the playlist link, and it still does not work.

![screen shot 2018-11-05 at 10 44 44 pm](https://user-images.githubusercontent.com/34232508/48005356-35233a80-e14e-11e8-89f1-31ae58f12a60.jpg)

**0.5.1:**

**OSX: 10.13 Mac sierra**

**Comment by akash-joshi** [comment]
The problem might be with the video, try other videos too

**Comment by necroshadow0** [comment]
![screen shot 2018-11-05 at 11 01 02 pm](https://user-images.githubusercontent.com/34232508/48005621-dc07d680-e14e-11e8-83d5-8a7d27d3debc.jpg)
![screen shot 2018-11-05 at 11 01 26 pm](https://user-images.githubusercontent.com/34232508/48005624-dca06d00-e14e-11e8-8efa-603496162427.jpg)
![screen shot 2018-11-05 at 11 01 19 pm](https://user-images.githubusercontent.com/34232508/48005626-ddd19a00-e14e-11e8-9b91-b343a0777961.jpg)

**Comment by The-Code-Monkey** [comment]
Cannot replicate can you send me the playlist and step by step of what you did to get the error

**Comment by The-Code-Monkey** [comment]
im guessing your using a https://youtu.be/aJOTlE1K90k link maybe its not allowing the redirect to youtube.com/watch=aJOTlE1K90k

**Comment by necroshadow0** [comment]
uhhh no
[here](https://www.youtube.com/playlist?list=PL6qJWDngc4uRbgKSyNXPdHASi2iFmPj4H)

**Comment by akash-joshi** [comment]
Hmm, the link submitted by @necroshadow0 and https://youtu.be/aJOTlE1K90k, both face this issue

**Comment by The-Code-Monkey** [comment]
I think it is something to do with auto redirects maybe being blocked by pennywise. also it seems to turn @necroshadow0 s link into an embed

**Comment by The-Code-Monkey** [comment]
{
  "ns": "yt",
  "el": "embedded",
  "cpn": "9kJLUCMThAbusfER",
  "docid": "vx2u5uUu3DE",
  "ver": 2,
  "cmt": "0",
  "fs": "0",
  "rt": "17.515",
  "euri": "",
  "lact": 1,
  "cl": "219577728",
  "mos": 0,
  "state": "80",
  "volume": 100,
  "c": "WEB_EMBEDDED_PLAYER",
  "cver": "20181101",
  "cplayer": "UNIPLAYER",
  "cbr": "Chrome",
  "cbrver": "70.0.3538.77",
  "cos": "Macintosh",
  "cosver": "10_14_0",
  "autoplay": "1",
  "hl": "en_GB",
  "cr": "GB",
  "len": "268",
  "fexp": "23710476,23721698,23721898,23740981,23741521,23744176,23751767,23752869,23754432,23755886,23755898,23756191,23758147,23760559,23762650,23762714,23762813,23763106,23763126,23763465,23764069,23764135,23767573,23767581,23768229,23768229,23769829,23770106,23770976,23771727,23772346,23772492,23773315,9441347,9445877,9449243,9456198,9474459,9485000",
  "list": "PL6qJWDngc4uRbgKSyNXPdHASi2iFmPj4H",
  "size": "581:1329:2",
  "inview": "1",
  "vct": "0.000",
  "vd": "NaN",
  "vpl": "",
  "vbu": "",
  "vpa": "1",
  "vsk": "0",
  "ven": "0",
  "vpr": "1",
  "vrs": "0",
  "vns": "0",
  "vec": "null",
  "vvol": "1",
  "debug_error": {
    "errorCode": "auth",
    "errorDetail": "0",
    "message": "Video unavailable",
    "subreason": ""
  },
  "relative_loudness": "NaN",
  "user_qual": "auto",
  "debug_videoId": "vx2u5uUu3DE",
  "0sz": false,
  "op": "",
  "yof": false,
  "dis": "",
  "gpu": "AMD_Radeon_Pro_555_OpenGL_Engine",
  "cgr": true,
  "debug_playbackQuality": "unknown",
  "debug_date": "Mon Nov 05 2018 15:29:32 GMT+0000 (GMT)"
}

the debug data from @necroshadow0 s playlist link

**Comment by necroshadow0** [comment]
huh
Or it's that mine is the playlist, not the video in the playlist

**Comment by The-Code-Monkey** [comment]
i think its because your hitting the link and youtube is seeing a desktop app and trying to do an embed for some reason

**Comment by necroshadow0** [comment]
okkkk

**Comment by necroshadow0** [comment]
that may be true...

**Comment by kamranahmedse** [comment]
Fixed in 0.6.0. The details on how to make it work [can be found here](https://github.com/kamranahmedse/pennywise#auto-embed-videos)

TL;DR turn off the `Edit > Embed Videos` option while running these videos.
